### ISSUE TITLE
Cannot Select Colours in Appearance Preferences

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#28500]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

If the button to access custom colours is pressed, navigation goes wrong and a second MuseScore window can appear.

Steps:
1. From the Edit menu select Preferences.
2. Select the Appearance tab.
3. Tab to the Background Colour radio button.
4. Arrow Right to the button showing the current colour
Actual output could be e.g. "Red 0 percent Green 0 percent Blue 0 percent button"
5. Press Space to activate the button.
Actual output: Speech says Basic Colors Alt+B
6. Press Space to activate the button.
Actual behaviour: nothing happens.
7. Arrow Up/Down
Actual behaviour: nothing happens.
8. Arrow Left/Right or press Tab:
Actual behaviour: focus returns to MuseScore, but the Color dialog is still there as well.

### Supporting files, videos and screenshots

N/A

### What is the latest version of MuseScore Studio where this issue is present?

4.5.2

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
